inc/class-sitemaps.php

```diff
@@ -1223,18 +1223,13 @@ function output() {
 	function sitemap_url( $url ) {
 
 		// Create a DateTime object date in the correct timezone
-		if ( isset( $url['mod'] ) ) {
-			$date = new DateTime( $url['mod'], new DateTimeZone( $this->get_timezone_string() ) );
-		}
-		else {
-			$date = new DateTime( date( 'y-m-d H:i:s' ), new DateTimeZone( $this->get_timezone_string() ) );
-		}
+		$date = $this->get_datetime_with_timezone( $url['mod'], $this->get_timezone_string() );
```

Comment:
There is no need to pass `$this->get_timezone_string()` because you can access it from `$this->get_datetime_with_timezone`